modules/installation-ibm-z-kvm-user-infra-machines-iso.adoc

```diff
@@ -48,14 +48,14 @@ $ qemu-img create -f qcow2 -F qcow2 -b /var/lib/libvirt/images/{source_rhcos_qem
 ----
 $ virt-install --noautoconsole \
    --connect qemu:///system \
-   --name {vm_name} \
-   --memory {memory} \
-   --vcpus {vcpus} \
-   --disk {disk} \
+   --name <vm_name> \
+   --memory <memory_mb> \
+   --vcpus <vcpus> \
+   --disk <disk> \
    --launchSecurity type="s390-pv" \ <1>
    --import \
-   --network network={network},mac={mac} \
-   --disk path={ign_file},format=raw,readonly=on,serial=ignition,startup_policy=optional <2>
+   --network network=<virt_network_parm>,mac=<mac_address> \
+   --disk path=<ign_file>,format=raw,readonly=on,serial=ignition,startup_policy=optional <2>
```

Comment:
I think that <ign_file> is the filename which is specified under step 3.
Therefore, it should be mentioned below under <2>.

Reply:
The step says "create the KVM guest nodes using the Ignition file..." Therefore, I don't think we need to add that information to the callout again.